clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/spring/client/annotation/AnnotationUtil.java

```diff
@@ -216,6 +219,17 @@ private static String getVariableName(final ParameterInfo param) {
     return param.getParameterName();
   }
 
+  private static boolean getVariableOptional(final ParameterInfo param) {
+    if (param.getParameterInfo().isAnnotationPresent(Variable.class)) {
+      final boolean optional = param.getParameterInfo().getAnnotation(Variable.class).optional();
+      LOG.trace("Extracting optional flag from Variable");
```

Comment:
IMO the log messages would convey much more information if they also contained the name of the parameter. As of now, such a log message is very hard to understand if it's found in the log.

Reply:
Good feedback. I would even extend it and say that it would be cool to know the whole class context (class/bean name, method name, parameter name). As this is a trace log, it can be verbose after all.